### ISSUE TITLE
Update to ID3 tag version 2.4

### ISSUE THREAD

**Issue by hildigerr** [myers/id3v2#6]
Some new frames have been added and some deprecated.
https://id3.org/id3v2.4.0-changes

**Comment by myers** [comment]
this is built on id3lib, which is has never been upgrade for the version 2.4 of id3v2 spec.  I recommend you using [mid3v2](https://mutagen.readthedocs.io/en/latest/man/mid3v2.html)

**Comment by hildigerr** [comment]
I greped for one of the new frames in id3lib and found it, and so assumed it had been updated. If I fork this and update it would you want pull request(s)?

**Comment by myers** [comment]
Yes please!

**Comment by myers** [comment]
What new frame are you looking for and what version of id3lib did you find it in?

**Comment by hildigerr** [comment]
Before opening this issue, I just searched for the first new tag from the list (ASPI) in the version of id3lib installed on my Arch machine (3.8.3).

I've noticed that Jetmore has a [fork](https://github.com/jetmore/id3v2) using TagLib to add id3v2.4 support. I pulled it into a branch on my fork to see if I could cherry pick and rebase it into a basis for switching to TagLib altogether. But now I'm thinking that If I'm going to use TagLib, I may as well start from scratch.

So, right now I'm trying to decide If I should do that or take a look into id3lib. I'm leaning toward starting from scratch, because I already have other applications that depend on TagLib anyway.

The other option I'm considering is creating an Arch package for [btag](https://github.com/fernandotcl/btag). I'm not using the other alternatives, such as Mutagen, due to the viral licensing restriction.
